### ISSUE TITLE
Add Publish button beside Done button

### ISSUE THREAD

**Issue by kevinj2016** [beaverbuilder/feature-requests#47]
Add a Publish button beside the Done button so that people don't have to click twice to publish a page. 

I could type control-P if my hand was on the keyboard, but my hand is always on the mouse when building pages and setting style options in modules.

It's an easy technical fix, has low risk of failure, doesn't require extensive testing, and would save customers millions of clicks.

**Comment by Dreamsync** [comment]
Yes. I’ve always been annoyed by the extra click

**Comment by zackpyle** [comment]
I disagree here. I think the extra step is fine and also prevents accidental publishing
